### ISSUE TITLE
somebody help me ? (freeze_graph) from where i wrong?

### ISSUE THREAD

**Issue by oerr** [davidsandberg/facenet#475]
tranned with inception v3 and generate input_graph pbtxt file like below

inception_train.py

# Save the model checkpoint periodically.
if step % 5000 == 0 or (step + 1) == FLAGS.max_steps:
checkpoint_path = os.path.join(FLAGS.train_dir, 'model.ckpt')
saver.save(sess, checkpoint_path, global_step=step)

pbName = "input_graph_{0}.pbtxt ".format(step)
tf.train.write_graph(sess.graph.as_graph_def(), FLAGS.train_dir, pbName, as_text=True)

- ckpt -> free_graph
bazel-bin/tensorflow/python/tools/freeze_graph \
--input_graph=/mnt/Data/ModelData/input_graph_19999.pbtxt \
--input_checkpoint=/mnt/Data/ModelData/model.ckpt-19999 \
--output_graph=/mnt/Data/ModelData/frozen_graph.pb \
--input_binary=False \
--output_node_names= ?

Question 1: what's the  output_node_names ?
-> i used "InceptionV3/Predictions/Reshape_1" but there has no name.

so i used "ExponentialMovingAverage" and can generate frozen_graph.pbtxt

but... i met error like below when i use label_image,

E tensorflow/examples/label_image/main.cc:294] 
Invalid argument: Input 0 of node ExponentialMovingAverage/AssignMovingAvg_387 was passed float 
from mixed_8x8x2048b/branch_pool/Conv/BatchNorm/moving_variance/ExponentialMovingAverage:0 incompatible with expected float_ref.

somebody help me?
